src/extension/index.ts

```diff
@@ -0,0 +1,20 @@
+import * as ProxyBindings from "./proxyBindings";
+
+export type ProxyAPI = typeof ProxyBindings;
+
+/**
+ * Initialize the window instance with bindings to proxy API functions
+ *
+ * @export
+ * @param { ProxyAPI } globalOrWindow
+ */
+export function bindToWindow(
+  globalOrWindow: typeof globalThis | Window,
+  name: string,
+) {
+  const instance = globalOrWindow as (typeof globalThis | Window) & {
+    [key in typeof name]: ProxyAPI;
+  };
+
+  instance[name] = ProxyBindings;
```

Comment:
Having ProxyApi/ProxyBindings referenced here kidna defeats the purpose of this being a util library.

Ideally, it would bind any passed object to global or window